### ISSUE TITLE
[Feature Request] Windows CLI terminal text displayed in the browser

### ISSUE THREAD

**Issue by xx88xx** [comfyanonymous/ComfyUI#2268]
I'm running the install remotely and don't have access to the original machine. It would be super useful to display the windows CLI terminal info  in browser to know if comfy has stalled or what not.

![image](https://github.com/comfyanonymous/ComfyUI/assets/60696375/ae4cc8c0-df53-493d-af31-d953b2c6a849)

Ie. This information ^

**Comment by ltdrdata** [comment]
Since logging is already being done in the Manager, it seems feasible to send the logs from the Manager to a display node.

**Comment by xx88xx** [comment]
> Since logging is already being done in the Manager, it seems feasible to send the logs from the Manager to a display node.

Yeah this makes complete sense. Hopefully it is an easy thing to add, would be immensely useful to keep things simple for users too.

**Comment by ltdrdata** [comment]
I added `Terminal Log (Manager)` node.
Update ComfyUI-Manager and try it.

**Comment by greeze** [comment]
> I added `Terminal Log (Manager)` node. Update ComfyUI-Manager and try it.

Perfect!

**Comment by xx88xx** [comment]
> I added `Terminal Log (Manager)` node. Update ComfyUI-Manager and try it.

@ltdrdata This works really well! exactly how I imagined. I just have one little QOL request (all good if it's too difficult to solve)

some of the coloured text and other text styles is getting sent through as raw code which makes a bit harder to read than the original terminal.
ie.
![image](https://github.com/comfyanonymous/ComfyUI/assets/60696375/db4d6c76-ab2a-4e9e-9e87-53faeae75af0)
vs
![image](https://github.com/comfyanonymous/ComfyUI/assets/60696375/cffd65ab-7867-47df-a193-2d786916bce0)

**Comment by xx88xx** [comment]
one issue, is that when comfyui is run in docker, this node doesn't function anymore. is there something I can do to get this working? docker is running in WSL on windows if that changes the situation at all.

**Comment by ltdrdata** [comment]
> @ltdrdata that node has been a god send. one issue I'm having now though, is that when comfyui is run in docker, this node doesn't function anymore. is there something I can do to get this working? docker is running in WSL on windows if that changes the situation at all.

I recommend comfyui-dev-utils.
